lib/matplotlib/axes/_axes.py

```diff
@@ -2825,6 +2825,12 @@ def xywhere(xs, ys, mask):
                          in cbook.safezip(x, xerr[1])]
             else:
                 # using list comps rather than arrays to preserve units
+                if not (len(xerr) == 1 or  # scalar
+                        # or symmetric errorbars, which must be a single list
```

Comment:
also, the comment about the list comprehensions is now separated from the list comprehensions.